content/docs/deploy/enterprise/changelog.mdx

```diff
@@ -7,9 +7,23 @@ toc_max_heading_level: 5
 
 # Changelog
 
+## 0.26.0
+
+### New
+
+In v0.26, we made some changes to the Enterprise Console's PPL builder. These updates include:
+
+- **Client certificate fingerprint criterion**: Enables you to specify a client certificate fingerprint value in the PPL builder
```

Comment:
This isn't quite right. Let me work on this and get back to you with an updated description of the new features.

Reply:
@kenjenkins ok. Just let me know whenever you're ready to move forward.

Reply:
I've pushed some changes here to try to describe this more precisely. Please feel free to edit further as you see fit.